src/goto-symex/goto_symex_state.cpp

```diff
@@ -780,3 +782,11 @@ void goto_symex_statet::print_backtrace(std::ostream &out) const
     }
   }
 }
+
+void goto_symex_statet::output_propagation_map(std::ostream &out)
```

Comment:
Documentation appreciated, otherwise there's a risk that this will be removed as unused in the next clean up.